server/protocols/protocols-imap4/src/test/resources/imapServerPlainAuthDisabledAndNotRequireSSL.xml

```diff
@@ -0,0 +1,14 @@
+<imapserver enabled="true">
+    <jmxName>imapserver</jmxName>
+    <bind>0.0.0.0:0</bind>
+    <tls socketTLS="false" startTLS="true">
+        <keystore>classpath://keystore</keystore>
+        <secret>james72laBalle</secret>
+        <provider>org.bouncycastle.jce.provider.BouncyCastleProvider</provider>
+        <algorithm>SunX509</algorithm>
+    </tls>
+    <auth>
+        <plainAuthEnabled>false</plainAuthEnabled>
```

Comment:
Why do we need this here?

Reply:
For `AuthenticationRequireSSL` test cases. But it has been disabled as temporary